util.go

```diff
@@ -1,73 +0,0 @@
-package gvm
-
-import (
-	"encoding/json"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"runtime"
-
-	"github.com/andrewkroh/gvm/common"
-	"github.com/pkg/errors"
-)
-
-func homeDir() (string, error) {
-	var homeDir string
-	if runtime.GOOS == "windows" {
-		homeDir = os.Getenv("USERPROFILE")
-	} else {
-		homeDir = os.Getenv("HOME")
-	}
-
-	if _, err := os.Stat(homeDir); err != nil {
-		return "", errors.Wrap(err, "failed to access home dir")
-	}
-
-	return homeDir, nil
-}
-
-func extractTo(to, file string) (string, error) {
-	tmpDir := to + ".tmp"
-	if err := os.Mkdir(tmpDir, 0755); err != nil {
-		return "", err
-	}
-	defer os.RemoveAll(tmpDir)
-
-	if err := common.Extract(file, tmpDir); err != nil {
-		return "", err
-	}
-
-	// Move into the final location.
-	if err := os.Rename(filepath.Join(tmpDir, "go"), to); err != nil {
```

Comment:
Could we keep this file as is but make it use the OS specific implementation of `copyDirectory()`? Otherwise there's a lot of duplication.